App.tsx

```diff
@@ -114,7 +114,7 @@ export default function App() {
                             </SearchBarContextProvider>
                           </WalletControlContextProvider>
                         </WalletsProvider>
-                      </DropdownsContextProvider>
+                      </ClickOutsideProvider>
```

Comment:
How about "DropdownsClickOutsideProvider"; maybe is too long, but I feel that with the new name the original intent "Dropdown" got lost.

Reply:
Thanks for the idea, yes I agree we can achieve new changes without renames. 
I can use:
~ClickOutsideProvider~ -> DropdownsContextProvider
~useClickOutside~ -> useDropdowns

So, we will have naming conventions as before, and it will be easier for devs to use this.

Reply:
Yep totally agree with that too 👍